test/LaunchDarkly.EventSource.Tests/EventSourceTests.cs

```diff
@@ -14,187 +13,95 @@ namespace LaunchDarkly.EventSource.Tests
     public class EventSourceTests
     {
         private readonly Uri _uri = new Uri("http://test.com");
-        private readonly TimeSpan _defaultReadTimeout = TimeSpan.FromSeconds(1);
-
-        [Fact]
-        public void Exponential_backoff_should_not_exceed_maximum()
-        {
-            TimeSpan max = TimeSpan.FromMilliseconds(30000);
-            ExponentialBackoffWithDecorrelation expo =
-                new ExponentialBackoffWithDecorrelation(TimeSpan.FromMilliseconds(1000), max);
-
-            var backoff = expo.GetNextBackOff();
-
-            Assert.True(backoff <= max);
-        }
-
-        [Fact]
-        public void Exponential_backoff_should_not_exceed_maximum_in_test_loop()
-        {
-            TimeSpan max = TimeSpan.FromMilliseconds(30000);
-
-            ExponentialBackoffWithDecorrelation expo =
-                new ExponentialBackoffWithDecorrelation(TimeSpan.FromMilliseconds(1000), max);
-
-            for (int i = 0; i < 100; i++)
-            {
-
-                var backoff = expo.GetNextBackOff();
-
-                Assert.True(backoff <= max);
-            }
-
-        }
-
-        [Fact]
-        public void Exponential_backoff_should_reset_when_reconnect_count_resets()
-        {
-            TimeSpan max = TimeSpan.FromMilliseconds(30000);
-
-            ExponentialBackoffWithDecorrelation expo =
-                new ExponentialBackoffWithDecorrelation(TimeSpan.FromMilliseconds(1000), max);
-
-            for (int i = 0; i < 100; i++)
-            {
-                var backoff = expo.GetNextBackOff();
-            }
-            expo.ResetReconnectAttemptCount();
-            // Backoffs use jitter, so assert that the reset backoff time isn't more than double the minimum
-            Assert.True(expo.GetNextBackOff() <= TimeSpan.FromMilliseconds(2000));
-        }
 
         [Fact]
         public async Task When_a_comment_SSE_is_received_then_a_comment_event_is_raised()
         {
-            // Arrange
-            var commentSent = ":";
+            var commentSent = ": hello";
 
             var handler = new StubMessageHandler();
+            handler.QueueResponse(StubResponse.StartStream(StreamAction.Write(commentSent + "\n\n")));
```

Comment:
For an explanation of why I added this "StubResponse/StreamAction" stuff instead of the simpler "QueueStringResponse", see StubHandler.cs.